test/unit/models/document_collection_test.rb

```diff
@@ -33,6 +33,17 @@ class DocumentCollectionTest < ActiveSupport::TestCase
     assert_valid build(:document_collection, body: nil)
   end
 
+  test "it should be able to change locale" do
+    doc_collection = create(:document_collection, groups: [])
+    assert doc_collection.locale_can_be_changed?
+  end
```

Comment:
We're already testing this in `test/unit/edition/translatable_test.rb`. Do we need the same test here?

Reply:
no we don't, thank you